pom.xml

```diff
@@ -285,11 +263,6 @@
                             <version>[1.9.0,1.10.0-!)</version>
                             <banSnapshots>true</banSnapshots>
                         </property>
```

Comment:
We should probably remove the above as well?

Without these lines, the versions updater will always try to update to the latest non-snapshot version of Core, which seems fine, though we may want to add this back in when 2.7.x is released, assuming we don't want to upgrade to it.

Reply:
Not sure.  That's out of scope for these changes I had to make.  I only remove the one I did because I removed the unnecessary "openmrsWebVersion" as something distinct from openmrsVersion.  I've never understood that...